### DISCUSSION TITLE
ODD/CD KSADS in FU3 confusion

### DISCUSSION THREAD

**Discussion by JaylenSantos** [now-i-know-my-abcd/docs#78] [Where in the World is…? (Help)] [answered]
I'm working with the ksads variables and I am trying to figure out why some folks were given the 1.0 instead of the 2.0 in FU3. Is this an error? There is a very small number of them. Additionally, does anyone know why less than 10 participants (parents) were given the ODD ksads 2.0? Seems odd that they would have gotten CD but not ODD? Thanks

**Comment by abaskinsommers** [comment, marked as answer]
For the 3yr FU parents were only given the CD, and not the ODD module. Decisions had to be made to account for time and the ODD module is on the following schedule: Baseline | 1yr fu | 2yr fu | NA | 4yr fu | NA |  NA | 7yr fu | 8yr fu. 

I will need to check on the 1.0 issue .

**Reply by JaylenSantos** [reply]
Thank you!

**Comment by abaskinsommers** [comment]
It seems that 1.0 was administered to these people as an error.
